Assets/VRTK/Source/Scripts/Utilities/VRTK_SharedMethods.cs

```diff
@@ -634,5 +614,43 @@ public static BuildTargetGroup[] GetValidBuildTargetGroups()
             }).ToArray();
         }
 #endif
+
+        /// <summary>
+        /// The FindEvenInactiveComponentsInLoadedScenes method searches active and inactive game objects in all
+        /// loaded scenes for components matching the type supplied. 
+        /// </summary>
+        /// <param name="searchAllScenes">If true, will search all loaded scenes, otherwise just the active scene.</param>
+        /// <param name="stopOnMatch">If true, will stop searching objects as soon as a match is found.</param>
+        /// <returns></returns>
+        private static IEnumerable<T> FindEvenInactiveComponentsInLoadedScenes<T>(bool searchAllScenes, bool stopOnMatch = false) where T : Component
```

Comment:
It was meant to be a shared method used by the other `FindEvenInactive(s)`. Since it has the slightly awkward parameter of stop early, i wasn't a big fan of it being public. I know private static methods are a little weird, but I prefer this to it being public i think